### PR TITLE
Fix typos in docs

### PR DESCRIPTION
**Pull request by eating247**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
-# [Welcome to The Odin Project a Open source Web development Curriculum](http://theodinproject.com)
-The Odin project is a open source community and curriculum for learning web development. Odin Project Students complete lessons which include links to the best free resources online and projects which they can use in their portfolio.
+# [Welcome to The Odin Project, an open source web development curriculum](http://theodinproject.com)
+The Odin Project is an open source community and curriculum for learning web development. Odin Project students complete lessons linking to the best free resources online, and projects which they can use in their portfolio.
```
Comment (arun1595) [inline README.md]: How about 'which they can _showcase_ in their portfolio.'?

Comment (codyloyd) [inline README.md]: showcase ++

Comment (arun1595) [inline README.md]: @eating247 

> Odin Project students complete lessons linking to the best free resources online, 

I personally feel this phrase can still be improved.

File: doc/linux_setup.md [hunk 2]
```diff
@@ -1,7 +1,7 @@
 **This guide assumes that you have already installed Ruby and rails. To get instructions on how to do that go through the installations project [here](http://www.theodinproject.com/web-development-101/installations)**
 
 ## Set up Postgres
-Skip this is you have already installed and setup a user for postgresql.
+Skip this if you have already installed and setup a user for postgresql.
```
Comment (arun1595) [inline doc/linux_setup.md]: Good catch :+1:

File: README.md (hunk 1 above)
Comment (leosoaivan) [inline README.md]: How about:

> Students of The Odin Project complete lessons that are constantly curated and updated with links to relevant resources.

I think we can drop the hyperbole (best) and I think the point that TOP is free is covered many times over already haha.

Comment (codyloyd) [inline README.md]: approve 👍

**Comment by eating247** [comment]
Updated the description per all of your suggestions! What do you think?

**Comment by codyloyd** [comment]
I dig it.
I say >MERGE<

let's see what Arun says ;)

**Comment by KevinMulhern** [comment]
@arun1595 once this gets you're stamp of approval we will :shipit:

**Pull request merged**